src/mixins/amalgamation-mixin.ts

```diff
@@ -241,6 +251,17 @@ export default class AmalgamationMixin extends Vue {
     )
   }
 
+  /**
+   * This business is draft task if the first task in the Todo List is still draft (or pending).
+   * @param business The business to check if is Draft Task or not.
+   */
+  isDraftTask (business: any): boolean {
+    return (
+      business.firstTask?.header.status === FilingStatus.DRAFT ||
+      business.firstTask?.header.status === FilingStatus.PENDING
+    )
```

Comment:
NB - Pending is it the filing as filed-and-paid but the payment failed. The user can retry or cancel the payment. If they cancel the payment then the filing goes back to Draft status.